examples/GraphicsTest.js

```diff
@@ -1,6 +1,5 @@
 var fs = require('fs');
-var Canvas = require('canvas');
-var Image = Canvas.Image;
+var { Canvas, Image } = require('canvas');

```

Comment:
Using this destructing syntax might limit older version of node. We can degrade it back to older syntax, if needed. But really a decision needs to be made on the minimum node version, and subsequently it should run on a CI test matrix that includes includes that minimum version of node.